vcf2cytosure.py

```diff
@@ -311,8 +311,8 @@ def parse_cn_coverages(args):
 		end = int(content[2])
 		coverage = float(content[5])
 		#first version
-		#yield CoverageRecord(chrom, start, end, -coverage)
-		yield CoverageRecord(chrom, start, end, coverage)
+		yield CoverageRecord(chrom, start, end, -coverage)
```

Comment:
Was this tested with Balsamic or with custom CNVkit runs? Anyway, easy enough to change, so merging for now and we can just update if needed.